pyproject.toml

```
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
```

Although this is distinct from the "CLI version" of `flit` we are using I think we should increase the lower bound here since we have been testing with 3.9 and `--only-deps` is new since 3.8 .
```suggestion
requires = ["flit_core >=3.9,<4"]
```